src/data/postgres/tables/ieee8021xProfiles.ts

```diff
@@ -160,7 +160,7 @@ export class IEEE8021xProfilesTable implements IIEEE8021xProfileTable {
       if (results.rowCount === 0) {
         return null
       }
-      return await this.getByName(item.profileName)
+      return await this.getByName(item.profileName, item.tenantId)
```

Comment:
did you confirm that this value is set from `req.tenantId` in the route? Otherwise it allows a user to assign the config to another tenantId

Reply:
hmm, im not seeing it. It looks like whatever comes from the DB is overridden from the request.

Reply:
I see your point. Just so I understand the scenario right, we should consider the case where `req.tenantId` (tenantId from auth header) could differ from `req.body.tenantId` (request body), this could allow changes to tenantId in the DB.